retrofit_gen/lib/parsed_info/parsed_info.dart

```diff
@@ -127,5 +127,6 @@ class WriteInfo {
 
   final List<Req> requests;
 
-  WriteInfo(this.name, this.basePath, this.basePathParams, this.baseMetaData, this.requests);
+  WriteInfo(this.name, this.basePath, this.basePathParams, this.baseMetaData,
```

Comment:
can you rename also this `this.baseMetaData` by `this.baseMetadata` my bad sorry :)

Reply:
Will do that.